core/src/main/java/com.panhuk.core/Fragments.kt

```diff
@@ -0,0 +1,7 @@
+package com.panhuk.core
+
+enum class Fragments {
```

Comment:
Will be better to rename this for ACTION or SCREEN. More abstract name not related to android Framework